README.md

```diff
@@ -1,2 +1,20 @@
-# KOBConnect
-Wallet Application for a KOB VON Network Member
+Title     | KOBConnect   
+----------| -------------
+Version   | _TBD_        
+## Abstract
+KOBConnect is the digital wallet application. The users can use it to store credentials and can access these stored credentials with ease. The wallet application is going to come with basic set of authentication to provide security for the user. The user can store anything useful.
+## Dependent Projects
+KOBWallet is a web application of KOBConnect app
+## Motivation
+It is actually a Physical wallet, but digital. It keeps all your digital credentials in one place. The user should be able to present these digital credentials whenever necessary. The user should also be able to decide what to share when necessary. 

```

Comment:
This point is not supposed to go here. You can talk about the current situation and how this wallet app is useful.